src/main/java/online/carsharing/entity/UserChatId.java

```diff
@@ -0,0 +1,21 @@
+package online.carsharing.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "users_chat_id")
+@Getter
+@Setter
+public class UserChatId {
+    @Id
+    @Column(name = "user_id", nullable = false)
+    private Long userId;
+
+    @Column(name = "chat_id", nullable = false)
+    private Long chatId;

```

Comment:
General question to the team, do we somehow name the columns in this way?

Reply:
Good point I can change user_id into user_telegram_id